kratos/python/add_processes_to_python.cpp

```diff
@@ -110,6 +111,40 @@ void CalculateEmbeddedVariableFromSkinArray(
     rDistProcess.CalculateEmbeddedVariableFromSkin(rVariable, rEmbeddedVariable);
 }
 
+std::unordered_map<int, std::vector<int> > GetNeighbourIds(
```

Comment:
I don't think this should be in "add_processes_to_python.cpp"

Reply:
I see your point, nevertheless i am not really sure of where to put it